src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -52,7 +52,7 @@ public int hashCode() {
      * Format state as text for viewing.
      */
     public String toString() {
-        return '[' + tagName + ']';
+        return (tagName + " ");
```

Comment:
I wonder why we would decide to change the toString() method at this point? It can surely cause inconvenience (conflicts) and, to my mind, should not be touched for the purpose of adding a new command